### ISSUE TITLE
orchestrate runner error

### ISSUE THREAD

**Issue by sokratisg** [jenkinsci/saltstack-plugin#91]
We've been using this plugin for its runner functionality for more than a year now.

Recently and after updating our salt master to latest stable release (2016.11.05), we have the following error:
```
salt/run/20170522113306534023/ret       {
    "_stamp": "2017-05-22T11:33:10.191089", 
    "fun": "runner.state.orchestrate", 
    "fun_args": [
        {
            "expr_form": "", 
            "mods": "orchestrate.do-something", 
            "pillar": "",
            "tgt": ""
        }
    ], 
    "jid": "20170522113306534023", 
    "return": "Exception occurred in runner state.orchestrate: Traceback (most recent call last):\n  File \"/usr/lib/python2.6/site-packages/salt/client/mixins.py\", line 395, in _low\n    data['return'] = self.functions[fun](*args, **kwargs)\nTypeError: orchestrate() got an unexpected keyword argument 'expr_form'\n", 
    "success": false, 
    "user": "jenkins"
}
```

Plugin configuration follows:
![2017-05-22 19_22_33](https://cloud.githubusercontent.com/assets/532528/26318717/b554cdb8-3f24-11e7-95b5-d54ba7622675.png)

Any suggestions?

**Comment by mchugh19** [comment]
Fixed in #93

**Comment by mchugh19** [comment]
If you are able, please try out [this beta release](https://github.com/jenkinsci/saltstack-plugin/releases/tag/v3.1.0-beta) incorporating this fix as well as corrections from #87 (fix for jenkins killing off pipeline job after 5 minutes).

**Comment by mchugh19** [comment]
This was corrected in release 3.1. Should be available on the mirrors in the next 24 hours.
